docs/docs/contributing/intellij-idea.md

```diff
@@ -20,40 +20,51 @@ its installed on your local machine). Otherwise, specify it by pressing *New*.
 
 ![](../../images/idea/idea-sdk.png "Select the JDK")
 
-On the final window we must select which modules we can import. Here, we are presented with the full list of SBT projects
+In some versions of IDEA, we must select which modules we can import. Here, we are presented with the full list of SBT projects
 that are defined in Dotty. You can either select all (expect performance degradation by IDEA if you select all) or
 select only the `dotty` module. In order to do that, unselect all modules and select on `dotty`. IDEA, then, automatically
 selects all the necessary dependencies and you press OK.
 
 ![](../../images/idea/idea-sbt.png "Select modules to import")
 
+In other versions of IDEA, no choice will be given. After the project loads, right click on `dotty` in the project explorer
+and select `Load/Unload Modules...`. Then select only `dotty`, accept when IDEA asks to add its dependencies and press OK.
+
 Running/Debugging
 -------
 
-To run the compiler you can do it either as an sbt command or via debugging the compiler.
-For the first option you can fire up sbt from the `Terminal` window of IDEA or you can do it externally.
+To run the compiler you can do it either as an sbt command or a shell script.
+For the first option you can fire up sbt from the `Terminal` window of IDEA or you from an external terminal.
```

Comment:
Although error messages are linkable and nice from my personal experience it's better to run it on an external terminal so personally I would give only one option here.

Reply:
Done.